src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -1,279 +1,400 @@
+    Picture: "https://www.google.com/capybara.png",
+  },
+  {
+    UserID: "def",
+    FirstName: "ken",
+    LastName: "ney",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+];
+
+
+// size - s,m,l
+function AuthorComponent({ user , size }) {
+
+  const fullName = user?.FirstName + " " + user?.LastName
+
+  const avatar = 
+      <Avatar
+      src={user?.Picture}
+      name={fullName}
+      size="md"
+      showBorder={true}
+      borderColor="black"
+      borderWidth="thick"
+     />
+
+  if (size === "s") {
     return (
-        <Card direction="row" width="50%">
-            <Avatar src={employee?.Picture} name={employee?.FirstName + " " + employee?.LastName} size='md' showBorder={true} borderColor='black' borderWidth='thick'/>
-            <CardBody>
-                <Text>{employee?.FirstName + " " + employee?.LastName}</Text>
-            </CardBody>
-        </Card>
+        <Popover trigger="hover">
+        <PopoverTrigger>
+          {avatar}
+        </PopoverTrigger>
+        <PopoverContent >
+          <PopoverArrow />
+          <PopoverCloseButton />
+          <PopoverBody>{fullName}</PopoverBody>
+        </PopoverContent>
+        </Popover>
     )
-}
-
-function SearchEmployeeTimesheet({employees, setSelected}) {
-    
-    const handleChange = (selectedOption) => {
-        setSelected(selectedOption);
-    }
-
-    const customStyles = {
-        control: (base) => ({
-          ...base,
-          flexDirection: 'row-reverse',
-        }),
-      }
-
-    const DropdownIndicator = (props) => {
-        return (
-          <components.DropdownIndicator {...props}>
-            <SearchIcon />
-          </components.DropdownIndicator>
-        );
-      };
-
-    // TODO: fix styling
-    // at the moment defaultValue is the first user in the employees array
-    // which is currently an invariant that matches the useState in Page
+  } else if(size === "m") {
+    return (
+      <Card width="50%">
+        {avatar}
+      <CardBody>
+        <Text>{fullName}</Text>
+      </CardBody>
+    </Card>
+    )
+  } else {
     return (
-        <div style={{width: '600px'}}>
-            <Select isSearchable={true} 
-            defaultValue={employees[0]} 
-            chakraStyles={customStyles} 
-            size="lg" 
-            options={employees} 
-            onChange={handleChange} 
-            components={{ DropdownIndicator }}
-            getOptionLabel={option =>`${option.FirstName + " " + option.LastName}`}
-            getOptionValue={option => `${option.FirstName + " " + option.LastName}`}/>
-        </div>
+      <Card width="50%">
+        {avatar}
+      <CardBody>
+        <Text>{fullName}</Text>
+        <Text>{user?.Email}</Text>
+      </CardBody>
+    </Card>
     )
+  }
 }
 
-export default function Page() {
-    //const today = moment(); 
-    const [selectedDate, setSelectedDate] = useState(moment().startOf('week').day(0)); 
+function ProfileCard({ employee }) {
```

Comment:
Same as with AuthorComponent per file, interface, and null/undefined case for employee.